python/private/py_console_script_binary.bzl

```diff
@@ -27,8 +27,7 @@ def _dist_info(pkg):
     rules_python does not know anything about the hub repos that the user has
     available.
 
-    NOTE: Works with `incompatible_generate_aliases` and without by assuming the
-    following formats:
+    NOTE: Works with assuming the following label formats:
```

Comment:
delete L31: `@pypi_pylint//:pkg` (nothing generates that format anymore, correct?)